resources/views/hr/job/create.blade.php

```diff
@@ -50,6 +50,15 @@
                             @endforeach
                         </select>
                     </div>
+                    <div class="col-md-3 form-group">
+                    <label for="start_date" class="fz-14 leading-none text-secondary mb-1">Start Date<strong class="text-danger"></strong></label>
+                    <input type="date" id="start_date" name="start_date" class="form-control">
+                    </div>
+
+                    <div class="col-md-3 form-group">
+                    <label for="end_date" class="fz-14 leading-none text-secondary mb-1">End Date<strong class="text-danger"></strong></label>
+                    <input type="date" id="end_date" name="end_date" class="form-control">
+                    </div>
```

Comment:
Please correct the formatting. Also why the `<strongclass="text-danger"></strong>` tag is used if it do not contain any text
```suggestion
                    <div class="col-md-3 form-group">
                        <label for="start_date" class="fz-14 leading-none text-secondary mb-1">Start Date<strong class="text-danger"></strong></label>
                        <input type="date" id="start_date" name="start_date" class="form-control">
                    </div>

                    <div class="col-md-3 form-group">
                        <label for="end_date" class="fz-14 leading-none text-secondary mb-1">End Date<strong class="text-danger"></strong></label>
                        <input type="date" id="end_date" name="end_date" class="form-control">
                    </div>
```